### ISSUE TITLE
Minor issues to be fixed from meeting 8th November 2023

### ISSUE THREAD

**Issue by andrewvanbreda** [BiologicalRecordsCentre/plantportal#96]

**Comment by andrewvanbreda** [comment]
Redirect to My Projects instead of All Projects after project creation.
Fixed.

**Comment by andrewvanbreda** [comment]
@sacrevert 
I have just tested the problem we saw earlier where Sam was able to manage membership even though she wasn't a project manager for your test project.
I have tested this behaviour and it is in fact correct.
This functionality is available to the project's manager, or a general website administrator.

**Comment by andrewvanbreda** [comment]
@sacrevert 
Fixed problem where user could click on map for spatial reference during data entry

**Comment by andrewvanbreda** [comment]
Changed following pages so the map zooms in automatically around the returned locations/samples (zoomMapToOutput option).
If a page isn't listed here, it probably already has this option on.

Standard Mode
My Samples
Plots Administration
Samples Administration

NPMS Mode
My Samples
Samples administration

**Comment by sacrevert** [comment]
Thanks @andrewvanbreda feel free to close if that is all the minor issues

**Comment by andrewvanbreda** [comment]
@sacrevert I will close once am sure have fixed the small issues that are on my list.

Another fixed, the My Samples page now has the site name on the samples grid, not just the occurrences grid.

**Comment by andrewvanbreda** [comment]
AVB note to myself:
Due to Drupal 10 rollouts, some changes might be lost (other issues too).
Change if need re-applying.

**Comment by andrewvanbreda** [comment]
Hi @sacrevert 
As discussed, the Management "Other" option is now a large text area on Standard Mode.
Let me know if you want this change to affect NPMS Mode and/or the NPMS website too.

**Comment by andrewvanbreda** [comment]
Hi @sacrevert 
Have carried out a test you requested in the meeting we had.
Anyone can use any plot on their project for data entry, but you can only make saves or deletions to your own plots.
This is working correctly.

**Comment by sacrevert** [comment]
> Hi @sacrevert As discussed, the Management "Other" option is now a large text area on Standard Mode. Let me know if you want this change to affect NPMS Mode and/or the NPMS website too.

Yes, please. Although only if it doesn't impact on our ability to extract historic data from this field via SQL queries (e.g. I wouldn't want to be in a situation where historic and new data were in different formats and required some operation to allow both to be retrieved in one SQL query)

**Comment by andrewvanbreda** [comment]
Hi @sacrevert,
You won't have those problems, it doesn't affect the database, it is just a visual control type.
I have also checked that the existing data is loaded into the field ok.
I have changed it on the following:
Plant Portal all 3 NPMS Mode data entry screens
Plant Portal - Tracking The Impact
NPMS website all 3 main data entry screens
(Plant Portal Standard Mode had already been changed)

**Comment by sacrevert** [comment]
Feel free to close if all dealt with here to you satisfaction

**Comment by andrewvanbreda** [comment]
@sacrevert 
OK, I will check if that is case when I can.

**Comment by andrewvanbreda** [comment]
Checked previous fixes are still present after Drupal 10 migration.

Also fixed problem where after joining a project, a list of available pages was displayed and the Project Homepage link would point to All Projects.
It was discussed to change this link to go to "List Plots", however there is a problem with this. NPMS Mode and Standard Mode use different project structures (and different URLs) and this link is designed to go to a single URL.
Instead I have removed the link as per screenshot (the user still has option to go to List Plots).

Closing issue now.

<img width="755" alt="Screen Shot 2023-11-29 at 19 14 32" src="https://github.com/BiologicalRecordsCentre/plantportal/assets/2463614/7abb78e0-b737-49e3-ac72-24d43f7400ea">
